src/_static/docker-compose-22.4.yml

```
@@ -160,7 +160,7 @@ services:
           - openvasd
 
   ospd-openvas:
-    image: greenbone/ospd-openvas:stable
+    image: registry.community.greenbone.net/community/ospd-openvas:edge
```

```suggestion
    image: registry.community.greenbone.net/community/ospd-openvas
```

Even if `edge` works, we do not want to use it in this compose file ... it should just be latest or stable I guess.